css/style.css

```
@@ -192,6 +193,57 @@ header .toggle {
   max-width: 600px;
 }
 
+.section3 .heading {
+  text-align: center;
+  font-size: 28px;
+  position: relative;
+  line-height: 1;
+}
+
+.section3 .heading span {
+  position: relative;
+  display: inline-block;
+  background-color: #ffffff;
+  width: 272px;
+}
+
+.section3 .heading:before {
+  position: absolute;
+  content: "";
+  top: 50%;
+  display: block;
+  border-top: 1px solid #eeeeee;
+  width: 100%;
+}
+
+.section3 .boxes {
+  margin-top: 40px;
+}
+
+.section3 .recommend .text {
+  font-size: 23px;
+  text-align: center;
+  line-height: 1.2;
+}
+
+.section3 .language {
+  margin-top: 100px;
+}
+
+.section3 .language .image {
+  width: 142px;
+  margin: auto;
+}
+
+.section3 .language .text {
+  padding-top: 10px;
+  border-top: solid 1px #eeeeee;
+  line-height: 1.5;
+}
+
+.section3 .text {
```

すみません、指摘に誤りがありました。
boxesのブロック内にはあてはまるので、下記のようにお願いします。
```suggestion
.section3 .boxes .text {
```

---

> すみません、指摘に誤りがありました。
> boxesのブロック内にはあてはまるので、下記のようにお願いします。

修正しました。
be4dff32858972b707128ba86ccbfc801ccf7223